website/screens/components/inset/code/examples/customSides.ts

```diff
@@ -0,0 +1,18 @@
+import { DxcInset, DxcStack } from "@dxc-technology/halstack-react";
+import Placeholder from "./Placeholder";
+
+const code = `() => {
+  return (
+    <DxcInset top="0.25rem" right="0.5rem" bottom="1rem" left="1.5rem">
+      <Placeholder height="large" />
+    </DxcInset>
```

Comment:
I think it is better to have several placeholders where one of them has an inset.

Reply:
done!